README.md

```diff
@@ -5,25 +5,29 @@ If you just want to visualize and parse data from the EmotiBit (for Windows/ mac
 If you want to modify the code(or build the tools in Linux), below are the requirements to build the project.  
 
 ## Requirements
-- OF 11 ~~https://openframeworks.cc/download/~~ NOTE: ofxEmotiBit is not yet compatible with 0.11.2. OF v0.11.0 can be downloaded here https://openframeworks.cc/versions/v0.11.0/
+
+#### openFrameworks
+
+Install openFrameworks 0.11.2 (vs2017_release) from the official [openFrameworks GitHub repository](https://github.com/openframeworks/openFrameworks/releases/tag/0.11.2).
+
 ### The following addons should be placed inside the OpenFrameworks addons folder:
-#### Note: If downloading zip instead of `git clone` be sure to remove `-master` from the folder name to avoid path discrepancies
-- ofxNetworkUtils - https://github.com/bakercp/ofxNetworkUtils
-- ofxOscilloscope - https://github.com/produceconsumerobot/ofxOscilloscope
-- ofxThreadedLogger - https://github.com/produceconsumerobot/ofxThreadedLogger
-- ofxBiquadFilter - https://github.com/mrbichel/ofxBiquadFilter
-- ofxJSON - https://github.com/jeffcrouse/ofxJSON
-- EmotiBit_XPlat_Utils - https://github.com/EmotiBit/EmotiBit_XPlat_Utils
-- ofxLSL - https://github.com/EmotiBit/ofxLSL
-  - _**Note:**_ for lsl support, if developing with visual studio, code should be compiled for x64
+#### Note: If downloading zip instead of `git clone` be sure to remove `-master`  or `-xxx-xxx` from the folder name to maintain correct path references. Move only the folder located inside each extracted add-on folder to `of_v0.11.2_vs2017_release\addons`.
```

Comment:
I think the "move only the folder located inside ..." sounds unclear. 
Maybe a better way to represent this would be to add a folder structure? Something like [this](https://gist.github.com/ryanflorence/daafb1e3cb8ad740b346#shared-modules) for the `addons` folder?